tests/test_upload.py

```diff
@@ -406,13 +407,17 @@ def upload_client_registration_fail():
     logger = logging.getLogger('upload_client')
     logger.addHandler(logging.StreamHandler())
     logger.setLevel(logging.DEBUG)
+
     # modify the client object used by upload_client so that replica registration fails
     class RegistrationFailureClient(Client):
+
```

Comment:
Are these spaces necessary?

Reply:
they were automatically added to meet the formatting rules - this file had probably not been touched since we enabled the formatting check, which is why it did not fail before